### PR TITLE
Pull request on branch 3

### PR DESCRIPTION
**Pull request by demjened**
Jeremy is changing the description to bump the resource x2.

### CODE REVIEW

**Comment by demjened** [comment]
Adding a comment, mentioning @demjened-test & @jeremy-dimasuay

**Comment by demjened** [comment]
Adding comment 2

**Comment by demjened** [comment]
Adding comment 3

File: data/myfile4.txt [hunk 1]
```diff
@@ -1 +1 @@
-adding
+modifying
```
Comment (demjened-test) [inline data/myfile4.txt]: Review comment 1

File: data/myfile5.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+adding
```
Comment (demjened-test) [inline data/myfile5.txt]: Review comment 2 mentioning @demjened

**Comment by demjened** [comment]
I agree.

**Comment by demjened** [comment]
Hey @jeremy-dimasuay, I'm mentioning you - please ignore.

File: data/myfile5.txt (hunk 2 above)
Comment (demjened) [inline data/myfile5.txt]: Review comment 3

**Comment by demjened** [comment]
test comment

**Comment by demjened** [comment]
test comment 2

**Comment by demjened** [comment]
Adding comment, mentioning @jeremy-dimasuay

File: data/myfile5.txt (hunk 2 above)
Comment (demjened) [inline data/myfile5.txt]: Adding review comment 4, mentioning @jeremy-dimasuay.

Comment (demjened) [inline data/myfile5.txt]: Review comment 5 mentioning @jeremy-dimasuay.

Comment (demjened) [inline data/myfile5.txt]: Review comment 6 mentioning @jeremy-dimasuay.

**Comment by demjened** [comment]
Hey-o

**Comment by demjened** [comment]
Test

**Comment by demjened** [comment]
Here's another comment.

**Comment by demjened** [comment]
mentioning @jeremy-dimasuay , please ignore

**Comment by demjened** [comment]
mentioning @demjened-test

File: data/myfile5.txt (hunk 2 above)
Comment (demjened) [reply data/myfile5.txt]: Mentioning @jeremy-dimasuay , please ignore

**Comment by demjened** [comment]
Comment

**Pull request closed without merging**